buildSrc/build.gradle.kts

```diff
@@ -85,7 +89,7 @@ dependencies {
     testCompile("com.nhaarman:mockito-kotlin:1.5.0")
 
     compile("org.pegdown:pegdown:1.6.0")
-    compile("org.jsoup:jsoup:1.6.3")
+    compile("org.jsoup:jsoup:1.11.2")
```

Comment:
:x:❓ why the jsoup upgrade in this PR?

Reply:
I used initially features that are only available in 1.11.2 that allow for DOM integration. I'm not using them anymore. I still thought it would be good to upgrade to a newer version as we are anyhow touching everything Jsoup related.